submissions/tosinIshola/html-forms/contact_form.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>CONTACT FORM</title>
+</head>
+<body>
+
+    <h1>Contact Form</h1>
+
+    <form action="#" method="post" enctype="multipart/form-data">
+
+        <fieldset>
+            <legend>Contact Form</legend>
+
+            <label for="firstname">First Name</label><br>
+            <input type="text" name="firstname" id="firstname"> <br>
+
+            <label for="lastname">Last Name</label> <br>
+            <input type="text" name="lastname" id="lastname"> <br>
+
+            <label for="email">Email</label> <br>
+            <input type="email" name="email" id="email" placeholder="Enter your email address"> <br>
+
+            Subject: <br/>
+            <textarea rows="3" cols="20" name="subject"></textarea> <br>
+
+            <label for= "comments">Message Input</label> <br>
+            <textarea name="comments" id="comments"  rows="12" cols="26" ></textarea> <br>
+
+            <button type="submit" onclick="alert('Thank you for contacting us, we will respond to you shortly')">Submit</button> <br>
```

Comment:
Welldone @gracioust. I can see that you are eager to show your coding ability but I think you should stick to the task instructions for now. You do not need the alert function you added to the submit button.